src/test/resources/et3ResponseDocument.json

```diff
@@ -0,0 +1 @@
+{"accessKey":"any","templateName":"ET3_0922 (1).docx","outputName":"ET3 Response.pdf","data":{"ethosCaseReference":null,"et3ResponseClaimantName":"claimant name","et3ResponseRespondentLegalName":"legal name","et3ResponseRespondentCompanyNumber":"1234","et3ResponseRespondentEmployerType":"employer type","et3ResponseRespondentPreferredTitle":null,"et3ResponseRespondentContactName":"contact name","respondentAddressLine1":"line1","respondentAddressLine2":"line2","respondentCity":"town","respondentCounty":null,"respondentPostcode":"postcode","et3ResponseDXAddress":null,"et3ResponsePhone":null,"et3ResponseContactPreference":null,"et3ResponseHearingRepresentative":null,"et3ResponseHearingRespondent":null,"et3ResponseEmploymentCount":"100","et3ResponseMultipleSites":null,"et3ResponseSiteEmploymentCount":"100","et3ResponseAcasAgree":null,"et3ResponseAcasAgreeReason":"acas agree reason","et3ResponseAreDatesCorrect":null,"et3ResponseEmploymentStartDate":"2022-01-01","et3ResponseEmploymentEndDate":"2022-01-02","et3ResponseEmploymentInformation":"employment information","et3ResponseContinuingEmployment":null,"et3ResponseIsJobTitleCorrect":null,"et3ResponseCorrectJobTitle":"fall guy","et3ResponseClaimantWeeklyHours":null,"et3ResponseClaimantCorrectHours":"168","et3ResponseEarningDetailsCorrect":null,"et3ResponsePayFrequency":null,"et3ResponsePayBeforeTax":"69000","et3ResponsePayTakehome":"25000","et3ResponseIsNoticeCorrect":null,"et3ResponseCorrectNoticeDetails":"notice details","et3ResponseIsPensionCorrect":null,"et3ResponsePensionCorrectDetails":"pension details","et3ResponseRespondentContestClaim":null,"et3ResponseContestClaimDetails":"contest claim","et3ResponseEmployerClaim":null,"et3ResponseEmployerClaimDetails":"ecc","et3ResponseRespondentSupportNeeded":null,"et3ResponseRespondentSupportDetails":"support details","titleMr":"■","titleMrs":"□","titleMiss":"□","titleMs":"□","titleOther":"□","titleText":null,"contactPost":"■","contactEmail":"□","hearingVideo":"□","hearingPhone":"■","siteYes":"■","siteNo":"□","acasYes":"□","acasNo":"■","datesYes":"□","datesNo":"■","datesNA":"□","continueYes":"□","continueNo":"■","continueNA":"□","jobYes":"□","jobNo":"■","jobNA":"□","hoursYes":"□","hoursNo":"■","hoursNA":"□","earnYes":"□","earnNo":"■","earnNA":"□","beforeWeekly":"■","beforeMonthly":"□","beforeAnnually":"□","takehomeWeekly":"■","takehomeMonthly":"□","takehomeAnnually":"□","noticeYes":"□","noticeNo":"□","noticeNA":"■","pensionYes":"□","pensionNo":"■","pensionNA":"□","contestYes":"■","contestNo":"□","eccYes":"□","helpYes":"■","helpNo":"□","helpNA":"□","repName":null,"repOrgName":null,"repAddressLine1":null,"repAddressLine2":null,"repTown":null,"repCounty":null,"repPostcode":null,"repDXNumber":null,"repPhoneNumber":null,"repMobileNumber":null,"repContactEmail":"□","repContactPost":"□","repEmailAddress":null,"repHearingVideo":"■","repHearingPhone":"□","et3ResponseRespondentMobile":null,"et3ResponseRespondentEmailAddress":null}}
```

Comment:
For these checked/unchecked characters, better we use a constant.  Only that I can see all kinds of problems coming in later if we don't

Reply:
Is that do-able in this json file? It's only read in as a string for a test